### ISSUE TITLE
Does not work with Rank Math SEO Plugin

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Checklists#122]
The checklist plugin does not work with Rank Math SEO Plugin.
Doesn’t allow to save new or earlier drafted posts. Only allows saving if it is an already published post.

![M8qI4Tv](https://user-images.githubusercontent.com/3868028/73264430-a70bf780-41ca-11ea-9fe6-ee687d46bf8c.jpg)

**Comment by andergmartins** [comment]
I have isolated the issue on the file: modules/checklists/assets/js/meta-box.js, on the line:

```javascript
wp.data.dispatch('core/editor').lockPostSaving('pp-checklists');
```

Since we use a timer, we are probably calling this action before the editor or other resource is fully loaded.
I just realized that calling the lockPostSaving with our plugin deactivated I can't reproduce the error in RankMath. I added a timer of 3 seconds to start our js script and that solved the issue. Not a real solution, but proves it is probably related to the order of things.

The solution for now is detect if Rank Math is loaded and add a timeout to the `PP_Checklists.init()` call.

**Comment by stevejburge** [comment]
Thanks @andergmartins. A workaround sounds good for now
